### DISCUSSION TITLE
New Blazor Web App Project Using Pocketbase - Check It Out!

### DISCUSSION THREAD

**Discussion by neozhu** [pocketbase/pocketbase#4979] [Show and tell]
Hello everyone,

I'm excited to share with you my new project - a Blazor web app built on top of Pocketbase. Pocketbase is a great backend solution that offers real-time data synchronization, file storage, and user authentication, making it a perfect fit for modern web applications.

Demo Site: https://blazorpocket.blazorserver.com/
GitHub Repo.: https://github.com/neozhu/BlazorPocketApp

![image](https://github.com/pocketbase/pocketbase/assets/1549611/3fb7be25-7620-4eff-a450-f2d818427196)
![image](https://github.com/pocketbase/pocketbase/assets/1549611/5a22118d-56c2-44e9-9029-62a5eb750b25)
![image](https://github.com/pocketbase/pocketbase/assets/1549611/f5158903-9893-4865-8cb9-e8fe71b6add6)

![image](https://github.com/pocketbase/pocketbase/assets/1549611/f4481e6c-1d3b-4d4c-b95d-06ee21472227)
![image](https://github.com/pocketbase/pocketbase/assets/1549611/645f8ee6-b592-49fc-aee0-c6060798aa4f)

**Comment by VictorioBerra** [comment]
PBcodegen thing is cool. @neozhu why do two separate docker containers instead of just 1?
